decidim-core/decidim-core.gemspec

```diff
@@ -61,11 +61,14 @@ Gem::Specification.new do |s|
   s.add_dependency "pg", "~> 1.4.0", "< 2"
   s.add_dependency "pg_search", "~> 2.2"
   s.add_dependency "premailer-rails", "~> 1.10"
-  # psych 4 has a breaking change, see https://stackoverflow.com/questions/71191685/visit-psych-nodes-alias-unknown-alias-default-psychbadalias
-  s.add_dependency "psych", "< 4"
+  # We need psych 4+ in order to use the `aliases: true` option,
+  # see https://stackoverflow.com/questions/71191685/visit-psych-nodes-alias-unknown-alias-default-psychbadalias
+  # Also note that we need Rails 6.1.5+,
+  # see https://gist.github.com/yahonda/2776d8d7b6ea7045359f38c10449937b
+  s.add_dependency "psych", "~> 4.0"
   s.add_dependency "rack", "~> 2.2", ">= 2.2.3"
   s.add_dependency "rack-attack", "~> 6.0"
-  s.add_dependency "rails", "~> 6.1.0"
+  s.add_dependency "rails", "~> 6.1.5"
```

Comment:
I think we are already using 6.1.6 , yet considering #10316 , i am ok with this

Reply:
@alecslupu It's the minimum version. 6.1.5 minimum or 6.1.X that is newer than that.